libhfuzz/instrument.c

```diff
@@ -123,6 +123,7 @@ static void initializeCmpFeedback(void) {
             _HF_CMP_BITMAP_FD, sizeof(cmpfeedback_t));
         return;
     }
+    memset(ret, 0, sizeof(cmpfeedback_t));
```

Comment:
mmap returns initialized memory (to 0'es). I don't think this is strictly necessary. Does the bug show without this line?